<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Compressing tiff file</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButtonAmFile">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Input Amira file</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelAmFile">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>0</y>
      <width>411</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>330</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Result:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelResult">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>330</y>
      <width>491</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>10</y>
      <width>371</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonCloseFile">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Close file</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelCloseFile">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>411</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_volume">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>250</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonResultFile">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Result file</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelResultFile">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>411</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_slice">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>100</y>
      <width>191</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLineEdit" name="lineEdit_intercept">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>60</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>456</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_slice">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>0</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Slice</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_axis">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>61</width>
       <height>80</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QRadioButton" name="radioButton_zaxis">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>51</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Z axis</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_yaxis">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>51</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Y axis</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_xaxis">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>51</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>X axis</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_sliceunits">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>50</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QRadioButton" name="radioButton_voxels">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>51</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>voxels</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_microns">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>41</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>um</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="pushButton_area">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>120</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Compute area</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_area">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_vessels">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>120</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Compute vessels</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_count">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>150</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_density">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>150</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_unitsarea">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>170</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>mm2</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_number">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>170</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>count</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_density">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>170</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>#/mm2</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_volume">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>220</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Compute volume</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_voxel">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>100</y>
      <width>131</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLineEdit" name="lineEdit_xvoxel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_yvoxel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_zvoxel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_voxel1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Voxel            Image</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_z">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>16</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Z</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_y">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>16</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Y</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_x">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>16</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_voxelsize2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Size (um)          Size</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_nx">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>40</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_ny">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>60</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_nz">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>80</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_unitsmm3">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>270</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>mm3</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>950</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonAmFile</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>amFileSelecter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>346</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonResultFile</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>resultFileSelecter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>55</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>115</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonCloseFile</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>closeFileSelecter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>41</x>
     <y>76</y>
    </hint>
    <hint type="destinationlabel">
     <x>183</x>
     <y>91</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_volume</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>computeVolume()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>188</x>
     <y>264</y>
    </hint>
    <hint type="destinationlabel">
     <x>69</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_xvoxel</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>voxelChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>171</x>
     <y>180</y>
    </hint>
    <hint type="destinationlabel">
     <x>84</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_yvoxel</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>voxelChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>179</x>
     <y>203</y>
    </hint>
    <hint type="destinationlabel">
     <x>120</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_zvoxel</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>voxelChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>174</x>
     <y>224</y>
    </hint>
    <hint type="destinationlabel">
     <x>137</x>
     <y>248</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_area</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>computeArea()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>382</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>269</x>
     <y>278</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_vessels</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>computeVessels()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>472</x>
     <y>262</y>
    </hint>
    <hint type="destinationlabel">
     <x>295</x>
     <y>305</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>go()</slot>
  <slot>amFileSelecter()</slot>
  <slot>closeFileSelecter()</slot>
  <slot>resultFileSelecter()</slot>
  <slot>computeVolume()</slot>
  <slot>checkReady()</slot>
  <slot>computeArea()</slot>
  <slot>computeVessels()</slot>
  <slot>voxelChanged()</slot>
 </slots>
</ui>
